src/spotifyaio/models.py

```diff
@@ -24,6 +24,7 @@ class DeviceType(StrEnum):
     SMARTPHONE = "Smartphone"
     SMARTWATCH = "Smartwatch"
     SPEAKER = "Speaker"
+    STB = "Set-Top Box"
```

Comment:
Other way around, STB is the value you get from Spotify, the first part is the name we give it